dist/landing.min.js

```diff
@@ -0,0 +1 @@
+function storeDataForProductPitchABTesting(){const o=location.pathname.toLowerCase();localStorage.setItem("productPitch",[[/\/(lp\/|)max($|\/.*)/,"MAX"],[/\/(lp\/|)cover($|\/.*)/,"COVER"],[/\/(lp\/|)vendre(-|\/).+/,"COVER"],[/\/.*/,"NONE"]].find(e=>e[0].test(o))[1])}let showError=!0;function displayError(){showError&&(showError=!1,alert("Merci de préciser votre numéro de rue."),setTimeout(()=>showError=!0,1e3))}function createCookieAndRedirect(e){var o={street_number:"short_name",route:"long_name",locality:"long_name",administrative_area_level_1:"short_name",country:"long_name",postal_code:"short_name"};"street_number"!=e.address_components[0].types[0]&&(document.cookie="street_number=1;path=/");for(const r of e.address_components){var t,n=r.types[0];o[n]&&(t=encodeURIComponent(r[o[n]]),document.cookie=n+`=${t};path=/`)}window.location.pathname="/offre/demande/v2"}function useFirstPrediction(o){var e=o.val();e&&(e.match(/^\d/)?(new google.maps.places.AutocompleteService).getPlacePredictions({input:e,componentRestrictions:{country:"fr"}},async e=>{e.length<=0||(o.val(e[0].description),createCookieAndRedirect((await(new google.maps.Geocoder).geocode({placeId:e[0].place_id})).results[0]))}):displayError())}function initAutoComplete(e,o){const t=$(e);e=$(o);const n=new google.maps.places.Autocomplete(t[0],{types:["address"],componentRestrictions:{country:"fr"}});n.setFields(["address_component"]),n.addListener("place_changed",()=>{var e;/^\d/.test(t.val())?(e=n.getPlace()).address_components&&createCookieAndRedirect(e):displayError()}),e.on("click",()=>(useFirstPrediction(t),!1)),t.keypress(e=>{if(13==e.which)return useFirstPrediction(t),!1})}$(function(){initAutoComplete("#autocomplete","#goingnext"),initAutoComplete("#autocomplete_2","#goingnext-2"),initAutoComplete("#autocomplete_3","#goingnext-3"),storeDataForProductPitchABTesting()});
```

Comment:
ah that's strange,`--mangle` only mangles variables and not our super-long references names like `storeDataForProductPitchABTesting`?

Reply:
Fixed